man/coap_oscore.txt.in

```diff
@@ -19,7 +19,8 @@ coap_delete_oscore_recipient,
 coap_new_client_session_oscore,
 coap_new_client_session_oscore_pki,
 coap_new_client_session_oscore_psk,
-coap_context_oscore_server
+coap_context_oscore_server,
+coap_register_oscore_context_handler
```

Comment:
As you may have discovered, you will need to add in an entry line into man/Makefile.am
````
        @echo ".so man3/coap_oscore.3" > coap_context_oscore_server.3
+      @echo ".so man3/coap_oscore.3" > coap_register_oscore_context_handler.3
        @echo ".so man3/coap_pdu_access.3" > coap_option_filter_set.3

````